packages/open-payments/src/client/requests.ts

```diff
@@ -65,12 +65,36 @@ export const get = async <T>(
 
 export const createAxiosInstance = (args: {
   requestTimeoutMs: number
+  privateKey: KeyLike
+  keyId: string
```

Comment:
This is the Rafiki instance's private key, right? Just wanted to clarify.

Reply:
correct